econet_base.yaml

```diff
@@ -97,6 +97,34 @@ text_sensor:
     request_mod: 1
     icon: "mdi:alert"
     entity_category: "diagnostic"
+  - platform: econet
+    name: "Alarm History 1"
+    id: alarm_history_1
+    sensor_datapoint: ALARMH01
```

Comment:
Is this populated for you? Do you mind testing the ALHISCLR? If it does can you add a button similar to ALRESET?